rpc/server.go

```diff
@@ -100,6 +102,66 @@ func (rs *RpcServer) registerHandlers() (err error) {
 				rs.client.Pay(obj.Channel, big.NewInt(int64(obj.Amount)))
 				return obj
 			})
+		case serde.ReceiveVoucherRequestMethod:
```

Comment:
This breaks substantially from the established pattern in this file.

It performs validation of received voucher in order to return an immediate result to the calling client.